lib/modules.txt

```diff
@@ -0,0 +1,43 @@
+require('assert');
```

Comment:
Ah, this needs to be removed. Was generated using `grep`, `sed`, etc. to track every file `BridgeClient` pulled in. We used it for finding the changes needed to be made to get `BridgeClient` isomorphic.